components/cascader/index.tsx

```diff
@@ -294,6 +294,9 @@ class Cascader extends React.Component<CascaderProps, CascaderState> {
   handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
     if (e.keyCode === KeyCode.BACKSPACE) {
       e.stopPropagation();
+    } else if (e.keyCode === 32) {
+      e.preventDefault();
+      (document.activeElement).click();
```

Comment:
And Cascader already support keyboard by `enter` to open.
This `handleKeyDown` only works on input element. When user use tab to focus, activeElement is the wrap div of cascader instead of input.

PS: This is not React style. Don't do this.

Reply:
If you want to support space, you can add PR to `rc-cascader`.